aws/rhel-10.0-nightly-x86_64/config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "user": "ec2-user",
+    "runnerArch": "amd64",
+    "prepareScript": "sudo rm -f /etc/yum.repos.d/*.repo; sudo tee /etc/yum.repos.d/rhel9internal.repo <<EOT\n[RHEL-9-NIGHTLY-BaseOS]\nname=baseos\nbaseurl=https://download.eng.brq.redhat.com/rhel-10/nightly/RHEL-10-Public-Beta/latest-RHEL-10/compose/BaseOS/x86_64/os/\nenabled=1\ngpgcheck=0\n\n[RHEL-9-NIGHTLY-AppStream]\nname=appstream\nbaseurl=https://download.eng.brq.redhat.com/rhel-10/nightly/RHEL-10-Public-Beta/latest-RHEL-10/compose/AppStream/x86_64/os/\nenabled=1\ngpgcheck=0\n\n[RHEL-9-NIGHTLY-CRB]\nname=crb\nbaseurl=https://download.eng.brq.redhat.com/rhel-10/nightly/RHEL-10-Public-Beta/latest-RHEL-10/compose/CRB/x86_64/os/\nenabled=1\ngpgcheck=0\nEOT"
```

Comment:
2 things:

1) Shouldn't we use download.devel to benefit from the proxying/mirroring capabilities instead of hard-coding a specific location
2) repository names should say 10, not 9